asab/library/service.py

```diff
@@ -382,9 +382,6 @@ async def _list(self, path, providers):
 	async def _read_disabled(self):
 		# `.disabled.yaml` is read from the first configured library
 		# It is applied on all libraries in the configuration.
-		if not self.is_ready():
-			raise LibraryNotReadyError("Library is not ready yet.")
```

Comment:
If I understand this issue properly, you need to wait here till the library is ready ... using while & async.sleep ... and maybe some kind of timeout.